### DISCUSSION TITLE
Why are scheduled Actions never executed?

### DISCUSSION THREAD

**Discussion by marchof** [community/community#26080] [Actions]
<p>Hi GitHub Community,</p>
<p>I automated several tasks with GitHub Actions. They are really powerful and flexible. I also tried a scheduled triggers like this on different repositories, for example <a href="https://github.com/marchof/io.javaalmanac.sandbox/blob/acbf70fc9040008977efb2a21d95716775676ec5/.github/workflows/cd.yml" rel="nofollow noopener">this one</a>:</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - master
    schedule:
      - cron: '0 0 * * *'
</code></pre>
<p>Unfortunately the workflow is never executed. I learned in this forum that scheduled triggers have some limitations, so I double checked:</p>
<ul>
<li>These are original repositories, no forks.</li>
<li>The schedule is defined on master branch</li>
<li>I deleted and created the workflow several times in subsequent commits</li>
</ul>
<p>It would be great to learn how to successfully trigger scheduled actions as these would greatly improve some of my FOSS dev tasks. Any input welcome!</p>
<p>Cheers,<br>
-marc</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I suspect what’s going wrong here is that you’re setting <code>schedule</code> as an attribute of <code>push</code> instead of <code>on</code>, where it is supposed to be. This may work better?</p>
<pre><code class="lang-yaml">on:
  push:
    branches:
      - master
  schedule:
    - cron: '0 0 * * *'
</code></pre>

**Comment by marchof** [comment]
<p>Indeed :man_facepalming:</p>
<p>I think yaml and me will never become friends.</p>
<p>Good catch <a class="mention" href="/lee-dohm">@lee-dohm</a>, thank you so much!</p>

**Comment by marchof** [comment]
<p>And now it actually works… thanks again. How can I send a virtual beer? :beer:</p>
